egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -183,70 +183,12 @@ def CheckArgs(args):
         args.add_lda = False
         warnings.warn("--add-lda is set to false as CNN layers are used.")
     
-    if args.final_mixture_window is not None:
-      if args.final_mixture_window <= 0 or rgs.final_mixture_window % 2 == 0:
-          raise Exception("--final-mixture-window should only be None or positive odd values.")
+    if args.final_mixture_window_length is not None:
+      if args.final_mixture_window_length <= 0 or rgs.final_mixture_window_length % 2 == 0:
+          raise Exception("--final-mixture-window-length supports only positive odd values.")
 
     return args
 
-def AddPerDimAffineLayer(config_lines, name, input, input_window):
-    filter_context = int((input_window - 1) / 2)
-    filter_input_splice_indexes = range(-1 * filter_context, filter_context + 1)
-    list = [('Offset({0}, {1})'.format(input['descriptor'], n) if n != 0 else input['descriptor']) for n in filter_input_splice_indexes]
-    filter_input_descriptor = 'Append({0})'.format(' , '.join(list))
-    filter_input_descriptor = {'descriptor':filter_input_descriptor,
-                               'dimension':len(filter_input_splice_indexes) * input['dimension']}
-
-
-    # add permute component to shuffle the feature columns of the Append
-    # descriptor output so that columns corresponding to the same feature index
-    # are contiguous add a block-affine component to collapse all the feature
-    # indexes across time steps into a single value
-    num_feats = input['dimension']
-    num_times = len(filter_input_splice_indexes)
-    column_map = []
-    for i in range(num_feats):
-        for j in range(num_times):
-            column_map.append(j * num_feats + i)
-    permuted_output_descriptor = nodes.AddPermuteLayer(config_lines,
-            name, filter_input_descriptor, column_map)
-
-    # add a block-affine component
-    output_descriptor = nodes.AddBlockAffineLayer(config_lines, name,
-                                                  permuted_output_descriptor,
-                                                  num_feats, num_feats)
-
-    return [output_descriptor, filter_context, filter_context]
-
-def AddMultiDimAffineLayer(config_lines, name, input, input_window, block_input_dim, block_output_dim):
```

Comment:
Let multidimaffinnelayer remain here.. it is still being tested and makes lot of assumptions about the architecture.